src/blocks/utils/SeamMeme/components/AppComposer.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+
+// components
+import Browser from "./Browser";
+import Editor from "./Editor";
+
+// utils
+import getMemes from "../utils/getMemes";
+
+// types
+import type { AppComposerProps, Meme } from "../types/types";
+
+// AppComposer
+const AppComposer = ({ model, done }: AppComposerProps) => {
+	// Browser component state
+	const [memes, setMemes] = useState<Meme[]>([]);
+	const [meme, setMeme] = useState<Meme | undefined>(undefined);
+
+	// Browser: Handler for setting a meme
+	const handleSetMeme = (meme: Meme | undefined) => {
+		setMeme(meme);
+	};
+
+	// Effect for loading memes
+	useEffect(() => {
+		const fetch = async () => {
+			const res = await getMemes();
+			setMemes(res);
+		};
+		fetch();
+	}, []);
+
+	return (
```

Comment:
General code styling, these should be 2spaces indents, rather than 4space tabs. fits the rest of the codebase and improves readability.